scss/_spinners.scss

```diff
@@ -1,36 +1,45 @@
 //
-// Rotating border
+// Spinners
 //
 
-@keyframes spinner-border {
-  to { transform: rotate(360deg); }
-}
-
-.spinner-border {
+%spinner {
```

Comment:
Watch out wit extends, you idea was probably to generate less code, but with this technique, you've added 2 extra selectors. I would prefer to see a mixin instead, but don't know if @mdo also agrees?